docs/installation/standard.md

```diff
@@ -24,8 +24,10 @@ You should download the bot using Git to make updating easier in the future.
 
 1. Clone the repository
 	```bash
-	git clone https://github.com/discord-tickets/bot.git
+	git clone https://github.com/discord-tickets/bot.git .
 	```
+	
+	Note: The . can be used to clone into the current directory...
 2. Find the most recent version with
```

Comment:
Formatting is broken.

![screenshot](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiZjE5MTlmZGUtMGVmNy00YjUyLTg4NGYtODcwZTM1NGY3NGU0IiwiZGVwbG95X2lkIjoiNjFmYmZkMjczNDI5ZDAwMDA3ZTIyNzYyIiwiaWQiOiJjY2Q4ZDVmNC1lMDgwLTRmMTItYjgwNS0xMGFiM2E3MmNiOTAifQ==)